_includes/templates/nav-about-us-media.html

```diff
@@ -0,0 +1,11 @@
+<a href="/the-bureau/">
+    <img src="../../static/img/nav-about-us-video.png"
```

Comment:
I don't think we should use the play arrow overlay if the video isn't going to play in the current window.